src/build.sh

```diff
@@ -303,7 +304,10 @@ for branch in ${BRANCH_NAME//,/ }; do
 
         DEBUG_LOG="$LOGS_DIR/$logsubdir/lineage-$los_ver-$builddate-$RELEASE_TYPE-$codename.log"
 
+        set +eu
+        # shellcheck source=/dev/null
```

Comment:
`# shellcheck source=/dev/null` is not needed here. (It was used to tell ShellCheck to not attempt to load/parse the target of a `source` command.)